sympy/tensor/tensor.py

```diff
@@ -38,6 +38,7 @@
 from collections import defaultdict
 import operator
 import itertools
+from sympy.core import Wild, WildFunction
```

Comment:
Ideally we import from the exact module within the library (this is not done everywhere in SymPy because this was not done so in the past).

Also keep the newline between the external and internal imports.

Reply:
I've changed it.